downstream/modules/platform/proc-controller-configure-system.adoc

```diff
@@ -0,0 +1,38 @@
+[id="controller-configure-system"]
+
+= Configuring system settings
+
+The *System* tab enables you to complete the following actions:
+
+* Define the base URL for the {ControllerName} host
+* Configure alerts
+* Enable activity capturing
+* Control visibility of users
+* Enable certain {ControllerName} features and functionality through a license file
+* Configure logging aggregation options
+
+.Procedure
+
+. From the navigation panel, select menu:Settings[].
+. From the *System* tab, choose from the following options:
+* *Miscellaneous System settings*: Enable activity streams, specify the default {ExecEnvShort}, define the base URL for the {ControllerName} host, enable {ControllerName} administration alerts, set user visibility, define analytics, specify usernames and passwords, and configure proxies.
+* *Miscellaneous Authentication settings*: Configure options associated with authentication methods (built-in or SSO), sessions (timeout, number of sessions logged in, tokens), and social authentication mapping.
+* *Logging settings*: Configure logging options based on the type you choose:
++
+image::ag-configure-aap-system-logging-types.png[Logging settings]
++
+For more information about each of the logging aggregation types, see the xref:assembly-controller-logging-aggregation[Logging and Aggregation] section.
+. Set the configurable options from the fields provided. 
+Click the tooltip image:tooltips-icon.png[Tool tip,15,15] icon next to the field that you need additional information about. 
```

Comment:
I can't see a tooltips-icon.png. There's question-circle.png ?

Reply:
I just added the tooltips-icon.png in this PR, I don't have a question-circle.png...